libs/shared/src/config/job-queue.ts

```diff
@@ -5,10 +5,12 @@ export enum JobTopicNameEnum {
   STANDARD = 'standard',
   WEB_SOCKETS = 'ws_socket_queue',
   WORKFLOW = 'trigger-handler',
+  SUBSCRIBER_PROCESS = 'subscriber-process',
```

Comment:
```suggestion
  PROCESS_SUBSCRIBER = 'process-subscriber',
```
Just a matter of meaning.